### PR TITLE
Fix everything with lasers

### PR DESCRIPTION
**Pull request by 2Cubed**
Using lasers, magic, and wizardry, I have fixed (and cleaned up) this example! :smiley:

### CODE REVIEW

**Comment by 2Cubed** [comment]
Sorry about that - forgot to configure Travis beforehand.

**Comment by Innectic** [comment]
\o/

**Comment by MattJustMatt** [comment]
CC'ing @connor4312 for review

File: chatty/socket.py [hunk 1]
```diff
@@ -1,77 +1,82 @@
...
-import random
-import json
 
-# A ws4py websocket wrapped in an Evented class.
-class EventedWebSocket(TornadoWebSocketClient, Evented):
-    def __init__(self, *args, **kwargs):
-        Evented.__init__(self)
-        TornadoWebSocketClient.__init__(self, *args, **kwargs)
+from tornado.websocket import websocket_connect
+from tornado.ioloop import IOLoop
```
Comment (connor4312) [inline chatty/socket.py:4]: While you're at it you may as well convert from Tornado to asyncio 😛

Comment (2Cubed) [inline chatty/socket.py:4]: Why? What's wrong with `tornado`?

Comment (connor4312) [inline chatty/socket.py:4]: With 3.x it seems like asyncio is the direction Python is moving for asynchronous capabilities, removing the need for 3rd party event frameworks like Tornado. But this is a bigger change, doesn't have to be in in this PR.

**Comment by connor4312** [comment]
Lgtm

File: chatty/socket.py (hunk 1 above)
Comment (2Cubed) [inline chatty/socket.py:4]: Fair enough. I might work on that Soon™. :smile: 
I also noticed the [`rewrite`](https://github.com/WatchBeam/beam-client-python/tree/rewritten) branch - would you like me to merge it in with what I just wrote?

**Pull request merged**